lang/nl.json

```diff
@@ -14,13 +14,24 @@
   "Unmute": "Geluid aan",
   "Playback Rate": "Weergavesnelheid",
   "Subtitles": "Ondertiteling",
-  "subtitles off": "Ondertiteling uit",
-  "Captions": "Ondertiteling",
-  "captions off": "Ondertiteling uit",
+  "subtitles off": "ondertiteling uit",
```

Comment:
Also, the menu has a text-transform css applied to it in the default skin, so, it would all be lower case. Though, that's also easily solved via a text transform. Ultimately, it doesn't matter *that* much what the translation files say specifically, as long as the keys match what's in the code, a user can set whatever they want via css.